ral-facilities/datagateway
# Bugfix/update count when filtering in search tabs #1006

## Description
Added a progress bar and additional code to update the count of the results when using filters in the search tabs.

## Testing instructions
Once opened, press the search button to get all the results in the search and use the filters available in the table column for advanced searching. Once the filter is used, a progress bar should appear while the results are being filtered and the results count on the tab labels should change to reflect the number of filtered results.


## Agile board tracking
Closes #1006
